test/expected/build/sap.ui.core-buildtime/dest/resources/sap/ui/core/Core-dbg.js

```diff
@@ -0,0 +1,5 @@
+/*!
+ * Some fancy copyright
+ */
+// replacement of $\{buildtime\} is only made in sap/ui/Global.js
+console.log('${buildtime}');
```

Comment:
According to our `.editorconfig`, there should always be a newline at the end of *.js files. Apparently your IDE doesn't seem to use this configuration. Please add newlines where missing, or my IDE will do it next time I save this file, creating unnecessary diffs 😉